apps/src/dance/constants.js

```diff
@@ -39,3 +39,18 @@ export const DancelabReservedWords = [
 
 export const ASSET_BASE =
   'https://curriculum.code.org/images/sprites/dance_20191106/';
+
+export const commonDanceBackgroundEffects =
```

Comment:
A few nits:
- are these expected to be used multiple places? Might be easier to keep where they're used if only used once?
- all caps for variable names?
- might be more readable to put them in an array (with one per line), then use them via array.join()

Regardless of of whether you do any of these, this is a nice bit of cleanup 👍

Reply:
I see that we have a duplicate list used in @tshaffercodeorg's PR https://github.com/code-dot-org/code-dot-org/pull/54299 in `HoC2023AiGenerateWeights.py` as @sanchitmalhotra126 pointed out.
I think I'll keep them in defined in this file for now to de-clutter `utils.ts`. Definitely should have one source of truth for these - task is created for this.

Also - I converted to arrays and used all caps. Thanks!